.github/workflows/chaosnet-docker.yml

```
@@ -52,13 +65,51 @@ jobs:
         uses: docker/build-push-action@v6
         with:
-          file: contrib/docker/chaosnet.Dockerfile
           context: .
+          target: chaosnet
           push: true
-          platforms: linux/amd64,linux/arm64
-          tags: ghcr.io/nibiruchain/chaosnet:${{ startsWith(github.ref, 'refs/tags/v') && steps.get_version.outputs.version || 'latest' }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
\ No newline at end of file
+          platforms: ${{ matrix.platform }}
```

actionlint (1.7.4)

property "platform" is not defined in object type {build: {platform: string; runner: string}}

(expression)